### PR TITLE
promisify and lipstickify

### PR DESCRIPTION
**Pull request by SamVerschueren**
Should we add AppVeyor tests for this module?

Had to refactor the tests because I couldn't do a `return` outside of a function.

### CODE REVIEW

**Comment by sindresorhus** [comment]
> Should we add AppVeyor tests for this module?

I remember trying. Wasted hours, but couldn't make it work.

File: readme.md [hunk 1]
```diff
@@ -13,15 +13,22 @@ $ npm install --save is-admin
...
+	//=> false
 });
 ```
 
 
+## API
+
+### isAdmin()
+
+Returns a Promise that resolves a boolean indicating if the process is running ad Administrator.
```
Comment (sindresorhus) [inline readme.md]:
```
Returns a Promise for a boolean indicating if the process is running as Administrator.
```

File: test.js [hunk 2]
```diff
@@ -1,31 +1,16 @@
...
-if (process.platform !== 'win32') {
-	test('non-win', function (t) {
-		t.plan(1);
-
-		childProcess.execFile(process.execPath, ['fixture.js'], {cwd: __dirname}, function (err) {
-			t.assert(err, err);
-		});
+if (process.platform === 'win32') {
+	test('normal user', t => {
+		t.throws(execa(process.execPath, ['fixture.js'], {cwd: __dirname}));
```
Comment (sindresorhus) [inline test.js]:
``` js
t.throws(execa('node', ['fixture.js'], {cwd: __dirname}));
```

`execa` handles the `node` resolution.

Comment (SamVerschueren) [inline test.js]: Cool! `execa` is so nice :).

**Comment by SamVerschueren** [comment]
This is untested on a windows machine though...

**Comment by sindresorhus** [comment]
Woot. Getting closer and closer to zero modules left now :)

Tried it on my Windows VM and tests are passing.

**Comment by SamVerschueren** [comment]
Yes, we're almost there :D

**Pull request merged**